src/y0/algorithm/identify/id_std.py

```diff
@@ -242,6 +242,7 @@ def line_6(identification: Identification) -> Expression:
     ranges = districts_without_treatments[0] - outcomes
     if not ranges:
         return expression
+    # TODO untested
```

Comment:
Sorry to be a pain but maybe we should get 100% coverage on ID given the other algorithms build on this

Reply:
this isn't related to the algorithms in this PR so save for later